### ISSUE TITLE
Would it make sense to apply batch normalisation along with spectral normalisation to the paper?

### ISSUE THREAD

**Issue by Nerdyvedi** [christiancosgrove/pytorch-spectral-normalization-gan#17]
In the paper [Inductive Guided Filter:Real Time Deep Image Matting with Weakly Annotated Masks on Mobile Devices](uhttps://arxiv.org/pdf/1905.06747.pdfrl), Both spectral normalisation and batch normalisation is being applied to the layers of the discriminator.

I was wondering would it make any sense to apply both, and If yes, How would it be implemented. They have not mentioned the order in which both the normalisations are applied .
